build-tasks/copy-config-json.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Converts configs/ to ./dist/build/configs/ and configs/recommended.json to ./dist/build/recommended.json.
+ */
+
+const mkdirp = require('mkdirp');
+const path = require('path');
+
+const { readDirectory, readFile, writeFile } = require('./common/files');
+
+const copyConfigFile = (configFileName, destinationFileName, newRulesDirectory) => {
+    const resolvedDestination = path.resolve(destinationFileName);
+    const data = readFile(path.resolve(configFileName));
+
+    writeFile(resolvedDestination, data.replace(/"rulesDirectory": \[(.*)\]/, `"rulesDirectory": ["${newRulesDirectory}"]`));
+};
+
+mkdirp.sync('./dist/build/configs');
+
+for (const configFileName of readDirectory('./configs')) {
+    copyConfigFile(`./configs/${configFileName}`, `./dist/build/configs/${configFileName}`, '../');
```

Comment:
Is is requirement to nest configurations in `config` folder?
I think that adding configuration to `tslint.json` without `config` will be a bit cleaner, however don't have strong opinion here.